docs/router.md

```diff
@@ -0,0 +1,204 @@
+# About Routers
+
+`gpii.express.router` is a subclass of [`gpii.express.middleware`](middleware.md) that wraps
+[the  Express `router` object](http://expressjs.com/en/4x/api.html#router), which is simply a container for a collection
+of middleware.  In our case, these are usually our child components that have the [`gpii.express.middleware`](middleware.md)
+grade.
+
+Routers can be nested, and you can even nest an instance of a router within itself.  The `path` variable for a given
+router module is combined with the paths of its parents, and that ultimately determines which URLs a given router will
+be given the chance to handle (see example below).  A `path` of `/` indicates that a router is dealing with the entire
+path.
+
+If multiple routers are registered for the same path, the first one to receive the request "wins".  As with middleware,
+you can control the order in which routers are added to the "chain" using
+[namespaces and priorities](http://docs.fluidproject.org/infusion/development/Priorities.html).
+The `path` option is one of two means by which a router can indicate that its scope is limited.  A router may also
```

Comment:
Actually, they are both required, but have defaults that indicate "any method" (`use`) or "any path" (`/`).  I will try to clarify to address this and the above "provides routing based purely on information present in the URL path" comment, but we should talk first.